external/src/main/java/com/redhat/cloud/policies/engine/AlertStarter.java

```diff
@@ -58,6 +62,30 @@ void startApp(@Observes StartupEvent startup) {
         initialize();
     }
 
+    private void printEnvVars() {
+        LOGGER.info("=== ENVIRONMENT VARIABLES - START ===");
```

Comment:
Since `ConfigProvider.getConfig().getValue` is the one failing, is there any way to print all the config values? (or ones prefixed with `engine` or so?)

Reply:
Yes, that is doable. I've sort of done similar in the ClowderConfigSource: https://github.com/RedHatInsights/clowder-quarkus-config-source/blob/7554c6108e2520a8e307955adabf9cd72d27058f/src/main/java/com/redhat/cloud/common/clowder/configsource/ClowderConfigSourceFactory.java#L39 my require a `ConfigSourceFactory` though (not sure)

Reply:
This should work (with proper whitelist filtering):
```
for (String key : ConfigProvider.getConfig().getPropertyNames()) {
    LOGGER.info(key + "=" + ConfigProvider.getConfig().getValue(key, String.class));
}
```